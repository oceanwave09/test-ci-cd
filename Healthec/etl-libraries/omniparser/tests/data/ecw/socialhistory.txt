apuid|RecordID|PatientId|PatientAccountNo|encounterId|EncounterDate|ProviderNPI|Ecwstructid|Question|QuestionID|Answers|AnswerID|ItemName|ItemID|CategoryName|CategoryID|MainCategoryName|MainCategoryID
280807|280807.17502.2459365.895|17502|17502|2459365|2022-10-17|1659584084|9|Had sex in the past 12 months (vaginal, oral, or anal)|895|Yes||Sexual hx|282119|Social History|10619|Annual Physical|10462
280807|280807.17502.2419095.895|17502|17502|2419095|2022-10-17|1679623896|9|Had sex in the past 12 months (vaginal, oral, or anal)|895|No||Sexual hx|282119|Social History|10619|Annual Physical|10462
280807|280807.17502.2517824.895|17502|17502|2517824|2022-10-17|1679623896|9|Had sex in the past 12 months (vaginal, oral, or anal)|895|No||Sexual hx|282119|Social History|10619|Annual Physical|10462
280807|280807.17502.2537450.895|17502|17502|2537450|2022-10-17|1679623896|9|Had sex in the past 12 months (vaginal, oral, or anal)|895|No||Sexual hx|282119|Social History|10619|Annual Physical|10462
280807|280807.17502.2540717.895|17502|17502|2540717|2022-10-17|1679623896|9|Had sex in the past 12 months (vaginal, oral, or anal)|895|No||Sexual hx|282119|Social History|10619|Annual Physical|10462
280807|280807.17502.2546108.895|17502|17502|2546108|2022-10-17|1679623896|9|Had sex in the past 12 months (vaginal, oral, or anal)|895|No||Sexual hx|282119|Social History|10619|Annual Physical|10462
280807|280807.17502.2551331.895|17502|17502|2551331|2022-10-17|1679623896|9|Had sex in the past 12 months (vaginal, oral, or anal)|895|No||Sexual hx|282119|Social History|10619|Annual Physical|10462
280807|280807.17502.2510419.895|17502|17502|2510419|2022-10-17|1679623896|9|Had sex in the past 12 months (vaginal, oral, or anal)|895|No||Sexual hx|282119|Social History|10619|Annual Physical|10462
280807|280807.17502.2527186.895|17502|17502|2527186|2022-10-17|1679623896|9|Had sex in the past 12 months (vaginal, oral, or anal)|895|No||Sexual hx|282119|Social History|10619|Annual Physical|10462
280807|280807.17502.2531535.895|17502|17502|2531535|2022-10-17|1679623896|9|Had sex in the past 12 months (vaginal, oral, or anal)|895|No||Sexual hx|282119|Social History|10619|Annual Physical|10462
280807|280807.17502.2544650.895|17502|17502|2544650|2022-10-17|1679623896|9|Had sex in the past 12 months (vaginal, oral, or anal)|895|No||Sexual hx|282119|Social History|10619|Annual Physical|10462
280807|280807.17502.2517560.895|17502|17502|2517560|2022-10-17|1679623896|9|Had sex in the past 12 months (vaginal, oral, or anal)|895|No||Sexual hx|282119|Social History|10619|Annual Physical|10462
280807|280807.17502.2542571.895|17502|17502|2542571|2022-10-17|1679623896|9|Had sex in the past 12 months (vaginal, oral, or anal)|895|No||Sexual hx|282119|Social History|10619|Annual Physical|10462
280807|280807.17502.2547790.895|17502|17502|2547790|2022-10-17|1679623896|9|Had sex in the past 12 months (vaginal, oral, or anal)|895|Yes||Sexual hx|282119|Social History|10619|Annual Physical|10462
280807|280807.17502.2538462.895|17502|17502|2538462|2022-10-17|1396385019|9|Had sex in the past 12 months (vaginal, oral, or anal)|895|No||Sexual hx|282119|Social History|10619|Annual Physical|10462
280807|280807.17502.2533209.895|17502|17502|2533209|2022-10-17|1396385019|9|Had sex in the past 12 months (vaginal, oral, or anal)|895|No||Sexual hx|282119|Social History|10619|Annual Physical|10462
280807|280807.17502.2531144.895|17502|17502|2531144|2022-10-17|1396385019|9|Had sex in the past 12 months (vaginal, oral, or anal)|895|No||Sexual hx|282119|Social History|10619|Annual Physical|10462
280807|280807.17502.2532849.895|17502|17502|2532849|2022-10-17|1396385019|9|Had sex in the past 12 months (vaginal, oral, or anal)|895|Yes||Sexual hx|282119|Social History|10619|Annual Physical|10462
280807|280807.17502.2552645.895|17502|17502|2552645|2022-10-17|1679623896|9|Had sex in the past 12 months (vaginal, oral, or anal)|895|No||Sexual hx|282119|Social History|10619|Annual Physical|10462
280807|280807.17502.2514075.895|17502|17502|2514075|2022-10-17|1417179151|9|Had sex in the past 12 months (vaginal, oral, or anal)|895|No||Sexual hx|282119|Social History|10619|Annual Physical|10462
280807|280807.17502.2506224.895|17502|17502|2506224|2022-10-17|1841223997|9|Had sex in the past 12 months (vaginal, oral, or anal)|895|No||Sexual hx|282119|Social History|10619|Annual Physical|10462
280807|280807.17502.2526959.895|17502|17502|2526959|2022-10-17|1841223997|9|Had sex in the past 12 months (vaginal, oral, or anal)|895|No||Sexual hx|282119|Social History|10619|Annual Physical|10462
280807|280807.17502.2539443.895|17502|17502|2539443|2022-10-17|1841223997|9|Had sex in the past 12 months (vaginal, oral, or anal)|895|No||Sexual hx|282119|Social History|10619|Annual Physical|10462
280807|280807.17502.2540894.895|17502|17502|2540894|2022-10-17|1841223997|9|Had sex in the past 12 months (vaginal, oral, or anal)|895|No||Sexual hx|282119|Social History|10619|Annual Physical|10462
280807|280807.17502.2550040.895|17502|17502|2550040|2022-10-17|1841223997|9|Had sex in the past 12 months (vaginal, oral, or anal)|895|No||Sexual hx|282119|Social History|10619|Annual Physical|10462
280807|280807.17502.2552051.895|17502|17502|2552051|2022-10-17|1841223997|9|Had sex in the past 12 months (vaginal, oral, or anal)|895|Yes||Sexual hx|282119|Social History|10619|Annual Physical|10462
280807|280807.17502.2398600.895|17502|17502|2398600|2022-10-17|1841223997|9|Had sex in the past 12 months (vaginal, oral, or anal)|895|No||Sexual hx|282119|Social History|10619|Annual Physical|10462
280807|280807.17502.2515651.895|17502|17502|2515651|2022-10-17|1841223997|9|Had sex in the past 12 months (vaginal, oral, or anal)|895|No||Sexual hx|282119|Social History|10619|Annual Physical|10462
280807|280807.17502.2526331.895|17502|17502|2526331|2022-10-17|1841223997|9|Had sex in the past 12 months (vaginal, oral, or anal)|895|No||Sexual hx|282119|Social History|10619|Annual Physical|10462
