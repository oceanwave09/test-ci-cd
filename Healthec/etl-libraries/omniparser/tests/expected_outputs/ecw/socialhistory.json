[
	{
		"answers": "Yes",
		"apuid": "280807",
		"category_id": "10619",
		"category_name": "Social History",
		"ecw_struct_id": "9",
		"encounter_date": "2022-10-17",
		"encounter_id": "2459365",
		"item_id": "282119",
		"item_name": "Sexual hx",
		"main_category_id": "10462",
		"main_category_name": "Annual Physical",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "67eaba81cfd50419ca1068ac5ce03825",
		"question": "Had sex in the past 12 months (vaginal, oral, or anal)",
		"question_id": "895",
		"record_id": "280807.17502.2459365.895",
		"row_id": "09a42a58-af40-4e95-aa8f-4ed4e47e6a7f"
	},
	{
		"answers": "No",
		"apuid": "280807",
		"category_id": "10619",
		"category_name": "Social History",
		"ecw_struct_id": "9",
		"encounter_date": "2022-10-17",
		"encounter_id": "2419095",
		"item_id": "282119",
		"item_name": "Sexual hx",
		"main_category_id": "10462",
		"main_category_name": "Annual Physical",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "985695c27e2cf3eea231a68c888cee60",
		"question": "Had sex in the past 12 months (vaginal, oral, or anal)",
		"question_id": "895",
		"record_id": "280807.17502.2419095.895",
		"row_id": "0febbfc5-bdd7-4c70-a79e-cd5d8f035b30"
	},
	{
		"answers": "No",
		"apuid": "280807",
		"category_id": "10619",
		"category_name": "Social History",
		"ecw_struct_id": "9",
		"encounter_date": "2022-10-17",
		"encounter_id": "2517824",
		"item_id": "282119",
		"item_name": "Sexual hx",
		"main_category_id": "10462",
		"main_category_name": "Annual Physical",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "985695c27e2cf3eea231a68c888cee60",
		"question": "Had sex in the past 12 months (vaginal, oral, or anal)",
		"question_id": "895",
		"record_id": "280807.17502.2517824.895",
		"row_id": "698d6f2b-e7d1-4efe-818e-5c4086ebf87b"
	},
	{
		"answers": "No",
		"apuid": "280807",
		"category_id": "10619",
		"category_name": "Social History",
		"ecw_struct_id": "9",
		"encounter_date": "2022-10-17",
		"encounter_id": "2537450",
		"item_id": "282119",
		"item_name": "Sexual hx",
		"main_category_id": "10462",
		"main_category_name": "Annual Physical",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "985695c27e2cf3eea231a68c888cee60",
		"question": "Had sex in the past 12 months (vaginal, oral, or anal)",
		"question_id": "895",
		"record_id": "280807.17502.2537450.895",
		"row_id": "faea19f9-a5b1-4a12-b96e-4d61230c58f1"
	},
	{
		"answers": "No",
		"apuid": "280807",
		"category_id": "10619",
		"category_name": "Social History",
		"ecw_struct_id": "9",
		"encounter_date": "2022-10-17",
		"encounter_id": "2540717",
		"item_id": "282119",
		"item_name": "Sexual hx",
		"main_category_id": "10462",
		"main_category_name": "Annual Physical",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "985695c27e2cf3eea231a68c888cee60",
		"question": "Had sex in the past 12 months (vaginal, oral, or anal)",
		"question_id": "895",
		"record_id": "280807.17502.2540717.895",
		"row_id": "464bcea6-0fa0-4de4-9cfe-273e250ffdc9"
	},
	{
		"answers": "No",
		"apuid": "280807",
		"category_id": "10619",
		"category_name": "Social History",
		"ecw_struct_id": "9",
		"encounter_date": "2022-10-17",
		"encounter_id": "2546108",
		"item_id": "282119",
		"item_name": "Sexual hx",
		"main_category_id": "10462",
		"main_category_name": "Annual Physical",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "985695c27e2cf3eea231a68c888cee60",
		"question": "Had sex in the past 12 months (vaginal, oral, or anal)",
		"question_id": "895",
		"record_id": "280807.17502.2546108.895",
		"row_id": "be065f01-77ce-48d0-9b7a-09bed7ead52a"
	},
	{
		"answers": "No",
		"apuid": "280807",
		"category_id": "10619",
		"category_name": "Social History",
		"ecw_struct_id": "9",
		"encounter_date": "2022-10-17",
		"encounter_id": "2551331",
		"item_id": "282119",
		"item_name": "Sexual hx",
		"main_category_id": "10462",
		"main_category_name": "Annual Physical",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "985695c27e2cf3eea231a68c888cee60",
		"question": "Had sex in the past 12 months (vaginal, oral, or anal)",
		"question_id": "895",
		"record_id": "280807.17502.2551331.895",
		"row_id": "9cc6e97d-a5a6-42b6-83cc-e81b291e8c21"
	},
	{
		"answers": "No",
		"apuid": "280807",
		"category_id": "10619",
		"category_name": "Social History",
		"ecw_struct_id": "9",
		"encounter_date": "2022-10-17",
		"encounter_id": "2510419",
		"item_id": "282119",
		"item_name": "Sexual hx",
		"main_category_id": "10462",
		"main_category_name": "Annual Physical",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "985695c27e2cf3eea231a68c888cee60",
		"question": "Had sex in the past 12 months (vaginal, oral, or anal)",
		"question_id": "895",
		"record_id": "280807.17502.2510419.895",
		"row_id": "54214b44-455a-4a8b-96aa-981afa3d4079"
	},
	{
		"answers": "No",
		"apuid": "280807",
		"category_id": "10619",
		"category_name": "Social History",
		"ecw_struct_id": "9",
		"encounter_date": "2022-10-17",
		"encounter_id": "2527186",
		"item_id": "282119",
		"item_name": "Sexual hx",
		"main_category_id": "10462",
		"main_category_name": "Annual Physical",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "985695c27e2cf3eea231a68c888cee60",
		"question": "Had sex in the past 12 months (vaginal, oral, or anal)",
		"question_id": "895",
		"record_id": "280807.17502.2527186.895",
		"row_id": "fc37d813-0527-4c3b-87b2-ef544f0f506c"
	},
	{
		"answers": "No",
		"apuid": "280807",
		"category_id": "10619",
		"category_name": "Social History",
		"ecw_struct_id": "9",
		"encounter_date": "2022-10-17",
		"encounter_id": "2531535",
		"item_id": "282119",
		"item_name": "Sexual hx",
		"main_category_id": "10462",
		"main_category_name": "Annual Physical",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "985695c27e2cf3eea231a68c888cee60",
		"question": "Had sex in the past 12 months (vaginal, oral, or anal)",
		"question_id": "895",
		"record_id": "280807.17502.2531535.895",
		"row_id": "94045199-bae4-415c-b80a-f5e7d4734266"
	},
	{
		"answers": "No",
		"apuid": "280807",
		"category_id": "10619",
		"category_name": "Social History",
		"ecw_struct_id": "9",
		"encounter_date": "2022-10-17",
		"encounter_id": "2544650",
		"item_id": "282119",
		"item_name": "Sexual hx",
		"main_category_id": "10462",
		"main_category_name": "Annual Physical",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "985695c27e2cf3eea231a68c888cee60",
		"question": "Had sex in the past 12 months (vaginal, oral, or anal)",
		"question_id": "895",
		"record_id": "280807.17502.2544650.895",
		"row_id": "89914a24-283f-4370-a1cd-706f18ee7141"
	},
	{
		"answers": "No",
		"apuid": "280807",
		"category_id": "10619",
		"category_name": "Social History",
		"ecw_struct_id": "9",
		"encounter_date": "2022-10-17",
		"encounter_id": "2517560",
		"item_id": "282119",
		"item_name": "Sexual hx",
		"main_category_id": "10462",
		"main_category_name": "Annual Physical",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "985695c27e2cf3eea231a68c888cee60",
		"question": "Had sex in the past 12 months (vaginal, oral, or anal)",
		"question_id": "895",
		"record_id": "280807.17502.2517560.895",
		"row_id": "9d852817-eb21-4c27-a0f7-0966c1d9bf22"
	},
	{
		"answers": "No",
		"apuid": "280807",
		"category_id": "10619",
		"category_name": "Social History",
		"ecw_struct_id": "9",
		"encounter_date": "2022-10-17",
		"encounter_id": "2542571",
		"item_id": "282119",
		"item_name": "Sexual hx",
		"main_category_id": "10462",
		"main_category_name": "Annual Physical",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "985695c27e2cf3eea231a68c888cee60",
		"question": "Had sex in the past 12 months (vaginal, oral, or anal)",
		"question_id": "895",
		"record_id": "280807.17502.2542571.895",
		"row_id": "a219016a-25fa-4f01-a8b9-9e172a14f237"
	},
	{
		"answers": "Yes",
		"apuid": "280807",
		"category_id": "10619",
		"category_name": "Social History",
		"ecw_struct_id": "9",
		"encounter_date": "2022-10-17",
		"encounter_id": "2547790",
		"item_id": "282119",
		"item_name": "Sexual hx",
		"main_category_id": "10462",
		"main_category_name": "Annual Physical",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "985695c27e2cf3eea231a68c888cee60",
		"question": "Had sex in the past 12 months (vaginal, oral, or anal)",
		"question_id": "895",
		"record_id": "280807.17502.2547790.895",
		"row_id": "41f4d825-daf2-4008-b115-20507373f64c"
	},
	{
		"answers": "No",
		"apuid": "280807",
		"category_id": "10619",
		"category_name": "Social History",
		"ecw_struct_id": "9",
		"encounter_date": "2022-10-17",
		"encounter_id": "2538462",
		"item_id": "282119",
		"item_name": "Sexual hx",
		"main_category_id": "10462",
		"main_category_name": "Annual Physical",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "1c369e06490b6b2cc59107d724bc80c8",
		"question": "Had sex in the past 12 months (vaginal, oral, or anal)",
		"question_id": "895",
		"record_id": "280807.17502.2538462.895",
		"row_id": "38fdfb72-4a31-4607-ad6f-7fbb07955e82"
	},
	{
		"answers": "No",
		"apuid": "280807",
		"category_id": "10619",
		"category_name": "Social History",
		"ecw_struct_id": "9",
		"encounter_date": "2022-10-17",
		"encounter_id": "2533209",
		"item_id": "282119",
		"item_name": "Sexual hx",
		"main_category_id": "10462",
		"main_category_name": "Annual Physical",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "1c369e06490b6b2cc59107d724bc80c8",
		"question": "Had sex in the past 12 months (vaginal, oral, or anal)",
		"question_id": "895",
		"record_id": "280807.17502.2533209.895",
		"row_id": "49d6001c-5b0d-4aff-8a37-e4c40070718b"
	},
	{
		"answers": "No",
		"apuid": "280807",
		"category_id": "10619",
		"category_name": "Social History",
		"ecw_struct_id": "9",
		"encounter_date": "2022-10-17",
		"encounter_id": "2531144",
		"item_id": "282119",
		"item_name": "Sexual hx",
		"main_category_id": "10462",
		"main_category_name": "Annual Physical",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "1c369e06490b6b2cc59107d724bc80c8",
		"question": "Had sex in the past 12 months (vaginal, oral, or anal)",
		"question_id": "895",
		"record_id": "280807.17502.2531144.895",
		"row_id": "54d28e9b-258a-43e3-9d8a-3023caa4aff4"
	},
	{
		"answers": "Yes",
		"apuid": "280807",
		"category_id": "10619",
		"category_name": "Social History",
		"ecw_struct_id": "9",
		"encounter_date": "2022-10-17",
		"encounter_id": "2532849",
		"item_id": "282119",
		"item_name": "Sexual hx",
		"main_category_id": "10462",
		"main_category_name": "Annual Physical",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "1c369e06490b6b2cc59107d724bc80c8",
		"question": "Had sex in the past 12 months (vaginal, oral, or anal)",
		"question_id": "895",
		"record_id": "280807.17502.2532849.895",
		"row_id": "f2690456-006e-456e-a9f0-6fdd382140db"
	},
	{
		"answers": "No",
		"apuid": "280807",
		"category_id": "10619",
		"category_name": "Social History",
		"ecw_struct_id": "9",
		"encounter_date": "2022-10-17",
		"encounter_id": "2552645",
		"item_id": "282119",
		"item_name": "Sexual hx",
		"main_category_id": "10462",
		"main_category_name": "Annual Physical",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "985695c27e2cf3eea231a68c888cee60",
		"question": "Had sex in the past 12 months (vaginal, oral, or anal)",
		"question_id": "895",
		"record_id": "280807.17502.2552645.895",
		"row_id": "b9953581-0dd3-45f6-b8ed-acf5758b9838"
	},
	{
		"answers": "No",
		"apuid": "280807",
		"category_id": "10619",
		"category_name": "Social History",
		"ecw_struct_id": "9",
		"encounter_date": "2022-10-17",
		"encounter_id": "2514075",
		"item_id": "282119",
		"item_name": "Sexual hx",
		"main_category_id": "10462",
		"main_category_name": "Annual Physical",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "cc8473289580a14bfaa0423d40f320e6",
		"question": "Had sex in the past 12 months (vaginal, oral, or anal)",
		"question_id": "895",
		"record_id": "280807.17502.2514075.895",
		"row_id": "72a5bff1-fe59-450e-a11d-46f6ac7b1efa"
	},
	{
		"answers": "No",
		"apuid": "280807",
		"category_id": "10619",
		"category_name": "Social History",
		"ecw_struct_id": "9",
		"encounter_date": "2022-10-17",
		"encounter_id": "2506224",
		"item_id": "282119",
		"item_name": "Sexual hx",
		"main_category_id": "10462",
		"main_category_name": "Annual Physical",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "f8146bd9f2775e71590bc4aa304ae03f",
		"question": "Had sex in the past 12 months (vaginal, oral, or anal)",
		"question_id": "895",
		"record_id": "280807.17502.2506224.895",
		"row_id": "b0d81944-e066-41ac-ba37-d17f36629103"
	},
	{
		"answers": "No",
		"apuid": "280807",
		"category_id": "10619",
		"category_name": "Social History",
		"ecw_struct_id": "9",
		"encounter_date": "2022-10-17",
		"encounter_id": "2526959",
		"item_id": "282119",
		"item_name": "Sexual hx",
		"main_category_id": "10462",
		"main_category_name": "Annual Physical",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "f8146bd9f2775e71590bc4aa304ae03f",
		"question": "Had sex in the past 12 months (vaginal, oral, or anal)",
		"question_id": "895",
		"record_id": "280807.17502.2526959.895",
		"row_id": "39a44376-d29a-4655-abb1-c84982b89222"
	},
	{
		"answers": "No",
		"apuid": "280807",
		"category_id": "10619",
		"category_name": "Social History",
		"ecw_struct_id": "9",
		"encounter_date": "2022-10-17",
		"encounter_id": "2539443",
		"item_id": "282119",
		"item_name": "Sexual hx",
		"main_category_id": "10462",
		"main_category_name": "Annual Physical",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "f8146bd9f2775e71590bc4aa304ae03f",
		"question": "Had sex in the past 12 months (vaginal, oral, or anal)",
		"question_id": "895",
		"record_id": "280807.17502.2539443.895",
		"row_id": "9edc2252-3e99-4977-b4ad-d98181f33c82"
	},
	{
		"answers": "No",
		"apuid": "280807",
		"category_id": "10619",
		"category_name": "Social History",
		"ecw_struct_id": "9",
		"encounter_date": "2022-10-17",
		"encounter_id": "2540894",
		"item_id": "282119",
		"item_name": "Sexual hx",
		"main_category_id": "10462",
		"main_category_name": "Annual Physical",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "f8146bd9f2775e71590bc4aa304ae03f",
		"question": "Had sex in the past 12 months (vaginal, oral, or anal)",
		"question_id": "895",
		"record_id": "280807.17502.2540894.895",
		"row_id": "39f18bf7-ca7c-46d1-92d8-33637809a2f1"
	},
	{
		"answers": "No",
		"apuid": "280807",
		"category_id": "10619",
		"category_name": "Social History",
		"ecw_struct_id": "9",
		"encounter_date": "2022-10-17",
		"encounter_id": "2550040",
		"item_id": "282119",
		"item_name": "Sexual hx",
		"main_category_id": "10462",
		"main_category_name": "Annual Physical",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "f8146bd9f2775e71590bc4aa304ae03f",
		"question": "Had sex in the past 12 months (vaginal, oral, or anal)",
		"question_id": "895",
		"record_id": "280807.17502.2550040.895",
		"row_id": "fd670014-f98f-4b04-9c66-da9dcfb8a636"
	},
	{
		"answers": "Yes",
		"apuid": "280807",
		"category_id": "10619",
		"category_name": "Social History",
		"ecw_struct_id": "9",
		"encounter_date": "2022-10-17",
		"encounter_id": "2552051",
		"item_id": "282119",
		"item_name": "Sexual hx",
		"main_category_id": "10462",
		"main_category_name": "Annual Physical",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "f8146bd9f2775e71590bc4aa304ae03f",
		"question": "Had sex in the past 12 months (vaginal, oral, or anal)",
		"question_id": "895",
		"record_id": "280807.17502.2552051.895",
		"row_id": "1af87457-3892-4861-b3af-f478398439b6"
	},
	{
		"answers": "No",
		"apuid": "280807",
		"category_id": "10619",
		"category_name": "Social History",
		"ecw_struct_id": "9",
		"encounter_date": "2022-10-17",
		"encounter_id": "2398600",
		"item_id": "282119",
		"item_name": "Sexual hx",
		"main_category_id": "10462",
		"main_category_name": "Annual Physical",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "f8146bd9f2775e71590bc4aa304ae03f",
		"question": "Had sex in the past 12 months (vaginal, oral, or anal)",
		"question_id": "895",
		"record_id": "280807.17502.2398600.895",
		"row_id": "da37daea-31fa-43c5-8ba7-6b43a4cdd042"
	},
	{
		"answers": "No",
		"apuid": "280807",
		"category_id": "10619",
		"category_name": "Social History",
		"ecw_struct_id": "9",
		"encounter_date": "2022-10-17",
		"encounter_id": "2515651",
		"item_id": "282119",
		"item_name": "Sexual hx",
		"main_category_id": "10462",
		"main_category_name": "Annual Physical",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "f8146bd9f2775e71590bc4aa304ae03f",
		"question": "Had sex in the past 12 months (vaginal, oral, or anal)",
		"question_id": "895",
		"record_id": "280807.17502.2515651.895",
		"row_id": "50323168-4132-480e-b1bc-fbc1bd151189"
	},
	{
		"answers": "No",
		"apuid": "280807",
		"category_id": "10619",
		"category_name": "Social History",
		"ecw_struct_id": "9",
		"encounter_date": "2022-10-17",
		"encounter_id": "2526331",
		"item_id": "282119",
		"item_name": "Sexual hx",
		"main_category_id": "10462",
		"main_category_name": "Annual Physical",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "f8146bd9f2775e71590bc4aa304ae03f",
		"question": "Had sex in the past 12 months (vaginal, oral, or anal)",
		"question_id": "895",
		"record_id": "280807.17502.2526331.895",
		"row_id": "b4e3cdd3-340c-4690-868c-f8fd65072fe7"
	}
]
